test/index.test.ts

```diff
@@ -0,0 +1,2 @@
+// require("./<NAME_CONTRACT_FIRST>/index.test");
```

Comment:
Чтобы все тесты проходили в нужном порядке тут наша точка входа.
в package.json у нас есть скрипт:
```
"test": "hardhat test ./test/index.test.ts"
```

И вызывать тесты через ``yarn test``

```suggestion
require("./Marketplace/index.test");
```